### ISSUE TITLE
False positive for `no-empty-attrs`

### ISSUE THREAD

**Issue by bdmac** [netguru/eslint-plugin-ember#24]
We have a local variable named `attr` and the `no-empty-attrs` rule is being triggered. Renaming the variable works but should not raise this.

```javascript
someArrayOfStrings.filter(function(attr) {
  return attr.underscore().endsWith('_id');
});
```

**Comment by michalsnik** [comment]
Yeah, makes sense. We should probably more strictly check this and scope the check to only model's properties.

**Comment by michalsnik** [comment]
I'll look into this soon 👍  For now you can disable this rule in this file or for this specific line:
```
/* eslint-disable-next-line ember/no-empty-attrs */
 return attr.underscore().endsWith('_id');
```

**Comment by michalsnik** [comment]
https://github.com/netguru/eslint-plugin-ember/releases/tag/v2.2.1
